src/openforms/registrations/admin.py

```diff
@@ -1,4 +1,11 @@
 from .fields import RegistrationBackendChoiceField
+from django.contrib import admin
+from .contrib.zgw_apis.models import TestPlugin
+from .contrib.zgw_apis.plugin import ZGWRegistration as zwgr
+from .contrib.stuf_zds.plugin import StufZDSRegistration as szr
+from .contrib.objects_api.plugin import ObjectsAPIRegistration as oar
+from .contrib.email.plugin import EmailRegistration as er 
+from .contrib.demo.plugin import DemoFailRegistration as dfr
```

Comment:
Imports from another package should use the full path. Since this is in the `registrations` package, all other paths should be absolute.

Reply:
full path import implemented